pkg/boost

```diff
@@ -17,19 +18,11 @@ dynamic-toolchain
 [build]
 # https://svn.boost.org/trac/boost/ticket/7847
 patch -p1 < "$K"/boost-execinfo.patch
-patch -p1 < "$K"/boost-endian.patch
```

Comment:
if the endian patch does no longer apply and isn't needed anymore (probably because GCC 6 defines __BYTE_ORDER__ by itself), then please remove it from KEEP too. (git rm KEEP/foo.patch)

Reply:
will do! I had just removed it on my end, forgot to git rm (:

and yeah I didn't check those two mirrors, I apologise.